<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editcourse</class>
 <widget class="QDialog" name="editcourse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Course</string>
  </property>
  <widget class="QPushButton" name="editCourseConfirm">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>192</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 150, 30);
color: white;</string>
   </property>
   <property name="text">
    <string>Edit Course</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelEditCourse">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>192</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 80, 64);
color: white</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>431</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Edit Course</string>
   </property>
   <widget class="QLineEdit" name="courseNameField">
    <property name="geometry">
     <rect>
      <x>107</x>
      <y>90</y>
      <width>300</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="courseCodeField">
    <property name="geometry">
     <rect>
      <x>107</x>
      <y>40</y>
      <width>300</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>46</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Course Code</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>97</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Course Name</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
